source/com.google.android.gms/play-services-games/Additions/IPendingResultExtensions.cs

```diff
@@ -225,96 +225,6 @@ public static async Task<ITurnBasedMultiplayerUpdateMatchResult> TakeTurnAsync (
         }
     }
 }
-namespace Android.Gms.Games.Quest
-{
-    public static partial class IQuestsExtensions
-    {
-        public static async Task<IQuestsAcceptQuestResult> AcceptAsync (this IQuests api, GoogleApiClient apiClient, string questId)
-        {
-            return (await api.Accept (apiClient, questId)).JavaCast<IQuestsAcceptQuestResult> ();
-        }
-        public static async Task<IQuestsClaimMilestoneResult> ClaimAsync (this IQuests api, GoogleApiClient apiClient, string questId, string milestoneId)
-        {
-            return (await api.Claim (apiClient, questId, milestoneId)).JavaCast<IQuestsClaimMilestoneResult> ();
-        }
-        public static async Task<IQuestsLoadQuestsResult> LoadAsync (this IQuests api, GoogleApiClient apiClient, int [] questSelectors, int sortOrder, bool forceReload)
-        {
-            return (await api.Load (apiClient, questSelectors, sortOrder, forceReload)).JavaCast<IQuestsLoadQuestsResult> ();
-        }
-        public static async Task<IQuestsLoadQuestsResult> LoadByIdsAsync (this IQuests api, GoogleApiClient apiClient, bool forceReload, params string [] questIds)
-        {
-            return (await api.LoadByIds (apiClient, forceReload, questIds)).JavaCast<IQuestsLoadQuestsResult> ();
-        }
-    }
-}
-namespace Android.Gms.Games.Request
-{
-    public static partial class IRequestsExtensions
-    {
-        public static async Task<IRequestsUpdateRequestsResult> AcceptRequestAsync (this IRequests api, GoogleApiClient apiClient, string requestId)
-        {
-            return (await api.AcceptRequest (apiClient, requestId)).JavaCast<IRequestsUpdateRequestsResult> ();
-        }

```

Comment:
I see this being removed, but the diff still has it? Is there some new automatic way of doing something?

Reply:
> Is there some new automatic way of doing something?

No. for sure.

I checked `IRequestsUpdateRequestsResult ` this and few other interfaces were removed. Not sure what is going on with diff. Could it be comparing with latest version (prerelease on nuget.org)

Reply:
Let us merge those few releases